tests/cocotb_tests/test_ral_read_write/Makefile

```diff
@@ -2,7 +2,7 @@ CWD=$(shell pwd)
 COCOTB_REDUCED_LOG_FMT = True
 SIM ?= icarus
 VERILOG_SOURCES =$(CWD)/clk.sv
-MODULE := test
+COCOTB_TEST_MODULES := test
```

Comment:
This isn't supported in cocotb < 2.0. This is likely breaking these tests. Leave it be.